app/cafe.py

```
@@ -0,0 +1,31 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        # Check vaccine key
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated")
+
+        vaccine = visitor["vaccine"]
+        expiration_date = vaccine.get("expiration_date")
+
+        if (
+            not isinstance(expiration_date, datetime.date)
+            or expiration_date < datetime.date.today()
+        ):
+            raise OutdatedVaccineError("Vaccine is outdated or invalid")
+
+        if not visitor.get("wearing_a_mask"):
+            raise NotWearingMaskError("Visitor is not wearing a mask")
```

This raises a mask error even if the visitor has no `wearing_a_mask` key. The checklist and examples require treating a friend as without a mask when `wearing_a_mask` is explicitly False. Quote from requirements: "Based on examples, a friend is considered without a mask if wearing_a_mask is explicitly False." To satisfy that, change the check to only raise when the key is present and falsy, e.g.: `if "wearing_a_mask" in visitor and not visitor.get("wearing_a_mask"):` then raise the NotWearingMaskError. This prevents treating a missing key as an explicit lack of mask.